docs/changelog.rst

```diff
@@ -74,6 +74,8 @@ Fix: ZpaTalker authenticate method prints the auth-token in the terminal which i
 Fix: ClientConnectorTalker download_service_status method renamed to list_download_service_status  and fixed response type return by api (by `Sergio Pereira <mailto:spereira@zscaler.com>`_)
 Fix: ClientConnectorTalker force_remove_devices and remove_devices method to remove company_id parameter and add userName and clientConnectorVersion parameters. (by`BrijeshShingadia<mailto:bshingadia@zscaler.com>`_)
 Fix: Removed company id requirement from zscaler client connector methods (by `Sergio Pereira <mailto:spereira@zscaler.com>`_)
+Fix: Migrate Portal Talker generic_delete to return bare response (by `Ryan Ulrick <mailto: rulrick@zscaler.com>`_)
```

Comment:
This need to ve under 5.0.1 as 5.0.0 has been already release.

Reply:
Changelog updated